cmd/nerdctl/container_commit_test.go

```diff
@@ -53,3 +56,56 @@ func TestCommit(t *testing.T) {
 		base.Cmd("rmi", testImage).Run()
 	}
 }
+
+func kubeCtlHelper(base *testutil.Base, args ...string) *testutil.Cmd {
+	base.T.Helper()
+	icmdCmd := icmd.Command("kubectl", args...)
+	icmdCmd.Env = base.Env
+	return &testutil.Cmd{
+		Cmd:  icmdCmd,
+		Base: base,
+	}
+}
+
+func TestKubeCommitPush(t *testing.T) {
+	t.Parallel()
+
+	base := testutil.NewBaseForKube(t)
+
+	var containerID string
+
+	setup := func() {
+		kubeCtlHelper(base, "run", "--image", "debian", "my-app", "--", "sleep", "Inf").
```

Comment:
Can we use testutil.BusyboxImage ?

Reply:
"my-app" should be prefixed with "nerdctl-", or probably moved to "nerdctl-test" namespace (`kubectl --namespace=nerdctl-test`)

Reply:
Moved the namespace to `nerdctl-test-k8s` for both Kubectl helper and BaseKube.

Reply:
Also moved to testutil.BusyboxImage

Reply:
Unless I misunderstand something, it does not look like kubectl namespaces are containerd namespaces.

```
kubectl get namespace
NAME                 STATUS   AGE
default              Active   5m22s
kube-node-lease      Active   5m22s
kube-public          Active   5m22s
kube-system          Active   5m22s
local-path-storage   Active   5m18s
nerdctl-test-k8s     Active   3m44s
```

```
nerdctl namespace list'
NAME      CONTAINERS    IMAGES    VOLUMES    LABELS
k8s.io    18            34        0
```